src/model/affiliation.ts

```diff
@@ -21,5 +21,11 @@ export class Affiliation extends BackmatterEntity {
 
   protected fromXML(_xml: Element): void {}
 
-  protected fromJSON(_json: JSONObject): void {}
+  protected fromJSON(json: JSONObject): void {
+    this._id = (json._id as string) || this.id;
+    this.label = json.label as string;
```

Comment:
for all of these string values, do we need to add a `|| ''` to catch for any undefined values?

Reply:
the validation works are post MVP. Apparently there're some complex rules we would need to implement but we agreed to assume all data is corrent on the client.